certbot/storage.py

```diff
@@ -38,13 +38,25 @@ def renewal_file_for_certname(config, certname):
             "{1}).".format(certname, path))
     return path
 
+def fullchain_for_renewal_conf(conf_file):
+    with open(conf_file) as f:
+        fullchain = re.findall('fullchain = (.*?)\n', f.read())[0]
```

Comment:
The file is a configuration object; look in this file for `configobj.ConfigObj(conf_file)`, `config_with_defaults`, `["fullchain"]`